search/src/commonMain/kotlin/com/islamversity/search/SearchState.kt

```diff
@@ -2,9 +2,11 @@ package com.islamversity.search
 
 import com.islamversity.core.mvi.BaseState
 import com.islamversity.core.mvi.BaseViewState
+import com.islamversity.domain.model.surah.SurahUIModel
 
 data class SearchState(
-    override val base: BaseState
+    override val base: BaseState,
+    val items : List<SurahUIModel>? = null
```

Comment:
please use non-null list with `emptyList()` as default value

Reply:
Ok, that is cause of rebase.